win32ss/user/ntuser/timer.c

```diff
@@ -19,10 +19,12 @@ static LONG TimeLast = 0;
 /* Windows 2000 has room for 32768 window-less timers */
 #define NUM_WINDOW_LESS_TIMERS   32768
 
+#define HINTINDEX_BEGIN_VALUE   1
+
 static PFAST_MUTEX    Mutex;
 static RTL_BITMAP     WindowLessTimersBitMap;
 static PVOID          WindowLessTimersBitMapBuffer;
-static ULONG          HintIndex = 1;
+static ULONG          HintIndex = HINTINDEX_BEGIN_VALUE;
```

Comment:
Nice: at least now the HintIndex is _really_ used (instead of just using it as a proxy for the constant "1").